lib/gems/pending/appliance_console/date_time_configuration.rb

```diff
@@ -12,10 +12,10 @@ class DateTimeConfiguration
 
     include ApplianceConsole::Logging
 
-    def initialize
-      @new_date         = nil
-      @new_time         = nil
-      @manual_time_sync = false
+    def initialize(config = {})
+      @new_date         = config.has_key(:date) ? config[:date] : nil
+      @new_time         = config.has_key(:time) ? config[:time] : nil
+      @manual_time_sync = config.has_key(:auto_sync) ? config[:auto_sync] : false
```

Comment:
For this one, if config is a Hash, then it's preferable to do `config.fetch(:auto_sync, false)`